web-client/public/js/grnsight.js

```diff
@@ -10,8 +10,8 @@ import { grnState } from "./grnstate";
 import { updateApp } from "./update-app";
 import { setupHandlers } from "./setup-handlers";
 
-setupHandlers(grnState);
-updateApp(grnState);
-
 container();
 upload(sliderObject, sliderGroupController, drawGraph, nodeColoringController);
+
+setupHandlers(grnState);
+updateApp(grnState);
```

Comment:
I suspect that if you do the MVC refactor correctly, then you won't have to make this change in order.

Reply:
Don't forget to undo this change if it isn't needed. Part of the whole "make your commits as small as possible" theme.